Creating documentation for [Get|Set]-CsTeamsSipDevicesConfiguration commandlets (#11216)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md

---
external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
Module Name: MicrosoftTeams
title: Set-CsTeamsSipDevicesConfiguration
author: anmandav
ms.author: anmandav
online version: https://learn.microsoft.com/powershell/module/teams/set-csteamssipdevicesconfiguration
schema: 2.0.0
---

# Set-CsTeamsSipDevicesConfiguration

## SYNOPSIS

This cmdlet is used to manage the organization-wide Teams SIP devices configuration.

## SYNTAX

```powershell
Set-CsTeamsSipDevicesConfiguration
   [-BulkSignIn <String>]
   [-WhatIf]
   [-Confirm]
   [<CommonParameters>]
```

## DESCRIPTION

> This cmdlet is used to manage the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to teams using Teams Sip Gateway.

Reviewer: ```suggestion
This cmdlet is used to manage the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to Teams using Teams Sip Gateway.
```

To execute the cmdlet, you need to hold a role within your organization such as Global Administrator, Teams Administrator, or Teams Communication Administrator.

## EXAMPLES

### Example 1

```powershell
Set-CsTeamsSipDevicesConfiguration -BulkSignIn "Enabled"
```
In this example, Bulk SignIn is set to Enabled across the organization.

### Example 2

```powershell
Set-CsTeamsSipDevicesConfiguration -BulkSignIn "Disabled"
```
In this example, Bulk SignIn is set to Disabled across the organization.

## PARAMETERS

### -BulkSignIn

> Indicates whether Bulk SingIn in to Teams SIP Devices is enabled or disabled for the Common Area Phone(CAP) accounts across the organization. Possible values are 'Enabled' and 'Disabled'.

Reviewer: ```suggestion
Indicates whether Bulk SingIn into Teams SIP devices is enabled or disabled for the common area phone (CAP) accounts across the organization. Possible values are **Enabled** and '**Disabled**.
```

---
external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
Module Name: MicrosoftTeams
title: Set-CsTeamsSipDevicesConfiguration
author: anmandav
ms.author: anmandav
online version: https://learn.microsoft.com/powershell/module/teams/set-csteamssipdevicesconfiguration
schema: 2.0.0
---

# Set-CsTeamsSipDevicesConfiguration

## SYNOPSIS

This cmdlet is used to manage the organization-wide Teams SIP devices configuration.

## SYNTAX

```powershell
Set-CsTeamsSipDevicesConfiguration
   [-BulkSignIn <String>]
   [-WhatIf]
   [-Confirm]
   [<CommonParameters>]
```

## DESCRIPTION

This cmdlet is used to manage the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to teams using Teams Sip Gateway.

You must be a Teams Service Administrator or a Global Administrator for your organization to run the cmdlet.

## EXAMPLES

### Example 1

> In this example, Bulk SignIn is set to Disabled across the organization.

Reviewer 2: Please move the description under the code block (example https://learn.microsoft.com/en-us/powershell/module/teams/new-teamchannel?view=teams-ps) and here it should say "Enabled"

Author: Done, moved and corrected it to Enabled
